deploy/crds/toolchain.dev.openshift.com_bannedusers.yaml

```diff
@@ -1,6 +1,11 @@
+
+---
```

Comment:
Previously when we tried to apply a CRD yaml with multiple manifests it failed. This is why we converted it to a single manifest yaml after generation. So, I assume it's not a problem anymore?

Reply:
I tested it locally and it seems to be working fine, so kept it there.
The latest operator-sdk also works with this format.